tests/pii_replacer/replacement/test_scope_consistency.py

```
@@ -0,0 +1,77 @@
+    PiiReplacementScope,
+)
+from nemo_safe_synthesizer.pii_replacer.entities import Config
+from nemo_safe_synthesizer.pii_replacer.replacement import run_replacement
+
+
+def _cfg() -> Config:
+    return Config(
+        locale="en_US",
+        random_seed=11,
+        persona_backend="faker",
+        sdg_pgms_src="/tmp",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)